tree_test.go

```diff
@@ -33,7 +33,8 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	mRand "math/rand"
+	mRandV1 "math/rand"
```

Comment:
can we not convert everything to v2?

Reply:
Unfortuantely no until we can use Go v1.23 which creates an API to do that exactly.